README.md

```
@@ -120,6 +120,171 @@ Examples:
+gh pr create --title "Feature: <name>" --body "..."
+
+# 4. Handle feedback PRAGMATICALLY
+# - Fix bugs in code (don't update specs)
+# - Address PR comments directly
+# - Update docs/checklists with lessons learned
+# - Specs are historical record, not living docs
+```
+
+### Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)